pkg/testutils/testutils.go

```diff
@@ -181,7 +182,8 @@ func WithBootstrapIP(ip string) func(*corev1.ConfigMap) {
 		if endpoints.Annotations == nil {
 			endpoints.Annotations = map[string]string{}
 		}
-		endpoints.Annotations[etcdcli.BootstrapIPAnnotationKey] = ip
+		// not relying on the constant from etcdcli.go here as this is introducing a cyclic dependency for its tests
+		endpoints.Annotations["alpha.installer.openshift.io/etcd-bootstrap"] = ip
```

Comment:
can't get it, would you kindly elaborate ?

Reply:
go modules can only depend in one direction and the testutils before this PR was depending on the etcdcli package and couldn't be used from there. Now that the etcdcli package uses testutils, you need to break the dependency somehow.

Reply:
I see, thanks so much 👍🏽